src/Avalonia.Base/Utilities/AvaloniaResourcesIndex.cs

```diff
@@ -66,20 +67,45 @@ private static void WriteIndex(BinaryWriter writer, List<AvaloniaResourcesIndexE
             }
         }
 
+        [Obsolete]
         public static void WriteResources(Stream output, List<(string Path, int Size, Func<Stream> Open)> resources)
```

Comment:
Since this method is part of public API, I made it obsolete, and added a new overload with AvaloniaResourcesEntry items.